IDEAS/Templates/Examples/ConstantAirFlowRecup.mo

```diff
@@ -11,8 +11,7 @@ model ConstantAirFlowRecup
   IDEAS.Templates.Structure.ThreeZone structure(
     redeclare package Medium = Medium)
     annotation (Placement(transformation(extent={{-80,-40},{-50,-20}})));
-  replaceable IDEAS.Templates.Ventilation.ConstantAirFlowRecup constantAirFlowRecup(
-    n=2.*structure.VZones)
+  replaceable IDEAS.Templates.Ventilation.ConstantAirFlowRecup constantAirFlowRecup(nZones=structure.nZones, n=2.*ones(structure.nZones))
   constrainedby IDEAS.Templates.Interfaces.BaseClasses.VentilationSystem(
     nZones=3,
```

Comment:
In this constrainedby declaration, the nZones is limited to 3. I guess it is better to remove this constraint.

Reply:
Indeed, the nZones is defined in the constantAirFlowRecup. I have removed it and pushed the changes

Reply:
I see that the unit tests do not pass for the example with no ventilation. That model extends from the constantAirFlowRecup and needs a specified nZones. I suggest reverting the commit (which I have already done).